helm_deploy/values-preprod.yaml

```diff
@@ -20,7 +20,7 @@ generic-service:
     DPS_HOME_PAGE_URL: "https://digital-preprod.prison.service.justice.gov.uk"
     PRISONER_PROFILE_URL: "https://prisoner-preprod.digital.prison.service.justice.gov.uk"
     ENVIRONMENT_NAME: PRE-PRODUCTION # Phase banner tag label
-    AUDIT_ENABLED: "false"
+    AUDIT_ENABLED: "true"
```

Comment:
Since we're making this true across the board, maybe removing the feature flag completely in this PR is a better change?